examples/service-sameness/sameness-test

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
```

Comment:
This script is awesome. I think that it's worth referencing it from the README to let users know that it will perform all the failover scenario steps and they can watch it run in real time.

Reply:
I have already mentioned it in the same README under the illustration section.

Reply:
:+1: I missed it the reference to the script during my review. All good.